src/types/search.ts

```diff
@@ -0,0 +1,5 @@
+export enum SearchFilters {
+  None = 0,
+  SharedByMe = 1 << 0,
```

Comment:
You really want to have byte operations here? As for me it's not needed

Reply:
at easiest way to define enum flags (2^N) which easiest way to store several flags in one value
https://stackoverflow.com/a/39359953

Reply:
Your example showing mostly bit operations - do you have any of them?
Straight numbers will be easier to understand and use

I understand that it easy to use, but It's not widely used thing and can be slightly confusing for newcomers. Maybe let's use just simple values if it's not used by any of your logic?

Reply:
@mikitabut . you can open several different documentation and will find this standard format of bit enum definition
e.g.
https://timdeschryver.dev/blog/flagged-enum-what-why-and-how#00826fd9a4f915cd83c47483a2a8f5cd
As result it would be constants after building. And also we are going to extand filters.
It would be useful for everybody including newcomers understand this format

Reply:
It will not be constants after building 
![image](https://github.com/epam/ai-dial-chat/assets/6322751/4224e6d5-ddb7-4a40-a90f-b1b7e19a2275)

Reply:
Still think that using bit operations is not needed here, but if you really want to use them ok, lets have them